heat/core/dndarray.py

```diff
@@ -1902,7 +1902,7 @@ def numpy(self):
         T1.numpy()
         """
        
-        return self.resplit(None)._DNDarray__array.cpu().numpy()
+        return self.resplit_(None)._DNDarray__array.cpu().numpy()
```

Comment:
This should remain resplit() as it is the numpy copy. The original dndarray should not be modified

Reply:
whoopsi, find and replace error ^^